Auto update failing for pv

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [pv](https://github.com/termux/termux-packages/blob/6fc1d3e0dcf875bea249313d8f04696a7e80e74f/packages/pv) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating pv [Current version: 1.9.0]
INFO: package being updated to 1.9.7.
Downloading http://www.ivarch.com/programs/sources/pv-1.9.7.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  3  372k    3 14013    0     0  46544      0  0:00:08 --:--:--  0:00:08 46554
100  372k  100  372k    0     0   617k      0 --:--:-- --:--:-- --:--:--  617k

You are about to commit these changes:

--------------------
diff --git a/packages/pv/build.sh b/packages/pv/build.sh
index 9075aa2471..40ee7faf83 100644
--- a/packages/pv/build.sh
+++ b/packages/pv/build.sh
@@ -2,8 +2,8 @@ TERMUX_PKG_HOMEPAGE=http://www.ivarch.com/programs/pv.shtml
 TERMUX_PKG_DESCRIPTION="Terminal-based tool for monitoring the progress of data through a pipeline"
 TERMUX_PKG_LICENSE="Artistic-License-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.9.0"
+TERMUX_PKG_VERSION="1.9.7"
 TERMUX_PKG_SRCURL=http://www.ivarch.com/programs/sources/pv-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=b38d69d7fc0785eb5eb3c57e8b12a7334f862047bf84b18f414365335399469a
+TERMUX_PKG_SHA256=6976a789ee1bbcfdc47732015b60589c1d0408892252dd7cbb44f43363a2f33a
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-ipc"
--------------------

bump(main/pv): 1.9.7


Not committing to Git!

INFO: Trying to build package.
INFO: Package pv exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building pv for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  89228      0 --:--:-- --:--:-- --:--:-- 89340
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  15478      0 --:--:-- --:--:-- --:--:-- 15716
gpg: Signature made Tue 03 Dec 2024 06:16:05 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0   301k      0 --:--:-- --:--:-- --:--:--  302k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  21408      0 --:--:-- --:--:-- --:--:-- 21921
gpg: Signature made Thu 28 Nov 2024 12:30:35 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0   273k      0 --:--:-- --:--:-- --:--:--  272k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  13135      0 --:--:-- --:--:-- --:--:-- 13222
gpg: Signature made Tue 03 Dec 2024 09:36:11 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading http://www.ivarch.com/programs/sources/pv-1.9.7.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 11  372k   11 42573    0     0   123k      0  0:00:03 --:--:--  0:00:03  123k
100  372k  100  372k    0     0   657k      0 --:--:-- --:--:-- --:--:--  656k
configure: WARNING: unrecognized options: --disable-rpath-hack, --enable-shared
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-linux-android-strip... llvm-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to enable C11 features... none needed
checking whether aarch64-linux-android-clang understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of aarch64-linux-android-clang... none
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking for an ANSI C-conforming const... yes
checking for _Bool... yes
checking for stdbool.h that conforms to C99... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for a sed that does not truncate output... /usr/bin/sed
checking whether NLS is requested... no
checking for msgfmt... /usr/bin/msgfmt
checking for gmsgfmt... /usr/bin/msgfmt
checking for xgettext... /usr/bin/xgettext
checking for msgmerge... /usr/bin/msgmerge
checking build system type... x86_64-pc-linux-gnu
checking host system type... aarch64-unknown-linux-android
checking for ld... ld.lld
checking if the linker (ld.lld) is GNU ld... yes
checking for shared library run path origin... done
checking 32-bit host C ABI... no
checking how to run the C preprocessor... aarch64-linux-android-cpp
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ELF binary format... yes
checking for the common suffixes of directories in the library search path... lib,lib,lib64
checking for CFPreferencesCopyAppValue... no
checking for CFLocaleCopyPreferredLanguages... no
checking whether to use NLS... no
checking for getopt_long... yes
checking for vsnprintf... yes
checking for strlcat... yes
checking for strtoul... yes
checking for memrchr... yes
checking for fdatasync... yes
checking for fpathconf... yes
checking for sysconf... yes
checking for posix_memalign... yes
checking for posix_fadvise... yes
checking for nanosleep... yes
checking for setitimer... yes
checking for setproctitle... no
checking for getopt.h... yes
checking for limits.h... yes
checking for wctype.h... yes
checking for termios.h... yes
checking for libgen.h... yes
checking for sys/ioctl.h... yes
checking for libintl.h... yes
checking for locale.h... yes
checking for sys/sysmacros.h... yes
checking for struct stat.st_blksize... yes
checking for library containing sqrtl... -lm
checking for math.h... yes
checking for sqrtl... yes
checking for library containing clock_gettime... none required
checking for alarm... yes
checking for basename... yes
checking for clock_gettime... yes
checking for dup2... yes
checking for getcwd... yes
checking for memcpy... yes
checking for memmove... yes
checking for memset... yes
checking for mkdir... yes
checking for select... yes
checking for setlocale... yes
checking for strchr... yes
checking for strerror... yes
checking for strrchr... yes
checking for strstr... yes
checking for fcntl.h... yes
checking for sys/file.h... yes
checking for sys/time.h... yes
checking for unistd.h... (cached) yes
checking for int32_t... yes
checking for mode_t... yes
checking for off_t... yes
checking for pid_t... yes
checking for size_t... yes
checking for ssize_t... yes
checking for uid_t in sys/types.h... yes
checking for struct stat.st_rdev... yes
checking siginfo_t provides a signal sender's PID... configure: error: in `/home/builder/.termux-build/pv/build':
configure: error: cannot run test program while cross compiling
See `config.log' for more details
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-12-03 12:21:46 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/12139314597">12139314597</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/pv/build.sh
+++ b/packages/pv/build.sh
@@ -1,9 +1,9 @@
-TERMUX_PKG_HOMEPAGE=http://www.ivarch.com/programs/pv.shtml
+TERMUX_PKG_HOMEPAGE=https://www.ivarch.com/programs/pv.shtml
 TERMUX_PKG_DESCRIPTION="Terminal-based tool for monitoring the progress of data through a pipeline"
-TERMUX_PKG_LICENSE="Artistic-License-2.0"
+TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.9.0"
-TERMUX_PKG_SRCURL=http://www.ivarch.com/programs/sources/pv-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=b38d69d7fc0785eb5eb3c57e8b12a7334f862047bf84b18f414365335399469a
+TERMUX_PKG_VERSION="1.9.15"
+TERMUX_PKG_SRCURL=https://www.ivarch.com/programs/sources/pv-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=e2b17564ab9eba1ec2caae285960cbf363b4401dffda191a60a0befe68e17dac
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-ipc"

